README.md

```diff
@@ -77,53 +68,73 @@ build/$(go env GOOS)/gotagger
 v0.4.0
 ```
 
-**Note**: The version reported may be different,
-depending on what unreleased changes exist.
+**Note**: The version reported may be different, depending on what unreleased
+changes exist.
 
-To tag a release,
-make any changes needed to prepare your project for releasing
-(ie. update the change log,
-merge any feature branches).
-Then create a "release" commit and run gotagger again:
+To tag a release, make any changes needed to prepare your project for releasing
+(ie. update the change log, merge any feature branches). Then create a "release"
+commit and run gotagger again:
 
 ```bash
 VERSION="$(gotagger)"
 git commit -m "release: $VERSION"
 gotagger -release
 ```
 
-You can now perform any release builds,
-push the tag to your central git repository,
-or any other post-release tasks.
+You can now perform any release builds, push the tag to your central git
+repository, or any other post-release tasks.
 
-`gotagger` can also push any tags it creates,
-by using the `-push` flag.
+`gotagger` can also push any tags it creates, by using the `-push` flag.
 
 ```bash
 gotagger -release -push
 ```
 
+#### Customizing Version Increments
+
+In some cases, you may wish to change which semantic version field should
+increment based on which commit type is provided. You can do this via
+a [config file](gotagger.json). The config file contains a mapping of commit
+type to semver increment and a default increment to use if it encounters an
+unknown commit type. For example:
+
+```json
+{
+  "incrementMappings": {
+    "release": "major",
+    "feat": "minor",
+    "fix": "patch"
+  },
+  "defaultIncrement": "none"
+}
+```
+
+Running:
+
+```bash
+gotagger -config ./gotagger.json
+```
+
+If no configuration is provided, Gotagger defaults to the current functionality,
+which is equivalent to what is defined in
+[gotagger.json](gotagger.json)
```

Comment:
Should gotagger default to reading a "gotagger.json" config file in the current directory if it exists, or is the explicit flag required?

Reply:
If gotagger.json is there, it'll use it. That behavior is overridden by the config file. If it doesn't exist, it'll take the default.